plugins/modules/net_tools/ipwcli_dns.py

```diff
@@ -45,7 +45,7 @@
     address:
         description:
             - The IP address for the A or AAAA record.
-            - Required for C(type=A) or C(type=AAAA)
+            - Required for I(type=A) or I(type=AAAA)
```

Comment:
Do you mind adding a period after the sentences as well, while we're at this? Same for the others as well :)
```suggestion
            - Required for I(type=A) or I(type=AAAA).
```

Reply:
done